gg-pac-team3-front/src/components/layout/Nav.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import logo from '../../assets/logos/blue.png';
+import styles from './nav.module.css';
+import { NavLink, Outlet} from "react-router-dom";
+
+function Nav(props){
+
+    const {isLoggedIn} = props;
+
+    const [navLinks, setNavLinks]  = useState({
+        link1: 'About',
+        link2: 'Login',
+        link3: 'Register'
+    });
```

Comment:
I don't see a reason to store navLinks in State, since they are entirely determined by `isLoggedIn` and do not change except when that prop changes. If we add more complex branching logic down the line this could change, but for now I would say lets not add unnecessary state management. Also, since this is ordered data, I would just store in an array. This might look something like:

```
const navLinks = loggedIn
    ? ['Dashboard', 'Account', 'Logout']
    : ['About', 'Login', 'Register'];
  ```
and in the return map over these like

```
          <ul>
            {navLinksNew.map((link) => (
              <li key={link}>
                <NavLink to={link.toLowerCase()}>{link}</NavLink>
              </li>
            ))}
          </ul>
```

(it's typical for routes to be lowercase just fyi)

Reply:
Makes sense. I wanted to try something like map earlier but because I had it in an obj it would not let me do a for  or if else statement in return